ChattoAdditions/Source/Chat Items/BaseMessage/Views/BaseMessageCollectionViewCell.swift

```diff
@@ -194,6 +203,13 @@ public class BaseMessageCollectionViewCell<BubbleViewType where BubbleViewType:U
         self.bubbleView.preferredMaxLayoutWidth = layoutModel.preferredMaxWidthForBubble
         self.bubbleView.layoutIfNeeded()
 
+        let avatarImageSize = baseStyle.getAvatarImageSize(self.messageViewModel)
+        if avatarImageSize != CGSize.zero {
+            self.avatarView.bma_rect = layoutModel.avatarViewFrame
+            self.avatarView.image = self.messageViewModel.avatarImage
+            self.avatarView.layoutIfNeeded()
```

Comment:
Does simply doing  self.avatarView.bma_rect = layoutModel.avatarViewFrame works fine? Can you move setting the image to updateViews?